.github/workflows/test_runner.yml

```diff
@@ -206,3 +218,15 @@ jobs:
     with:
       safe-checkout: ${{ needs.check-tag.outputs.checkout-sha }}
     secrets: inherit
+
+  # This test depends on all blocking tests and indicates whether they all suceeded.
+  all_blocking_tests:
+    name: All Blocking Tests
+    needs: [bazel, cpp, java, python, ruby, php, php-ext, csharp, objectivec, rust, upb, staleness]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check test results
+        run: "${{ !contains(join(needs.*.result, ' '), 'failure') && !contains(join(needs.*.result, ' '), 'cancelled') && !contains(join(needs.*.result, ' '), 'skipped') }}"
```

Comment:
Won't staleness tests be skipped on 'scheduled' runs?  Unfortunately we don't have any way to truly test that, but you could update the `if:` above to include `workflow_dispatch` and see what happens when you manually trigger this

Reply:
Yea I can't think of a case where a skip would be a failure, especially at the top level like this.  We could probably just ignore skipped test suites